### PR TITLE
WRQ-30274: Prepare for merging `feature/enact_v5` to `develop` branch

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We will merge `feature/enact_v5` branch to `develop` to release next major alpha version of sandstone.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Merged `feature/enact_v5` branch to `develop`
- Modified `npm-shrinkwrap.json` in sampler
- Changed ubuntu dist in travis.yml from `focal` to `jammy`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-30274

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Review by seunghoh — CHANGES_REQUESTED** [review]
Please fill in the PR description

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ install:
     - npm install
     - npm link
     - popd
-    - git clone --branch=develop --depth 1 https://github.com/enactjs/enact ../enact
+    - git clone --branch=feature/enact_v5 --depth 1 https://github.com/enactjs/enact ../enact
```
Comment (MikyungKim) [inline .travis.yml:20]:
```suggestion
    - git clone --branch=develop --depth 1 https://github.com/enactjs/enact ../enact
```

File: samples/event-logger/package.json [hunk 2]
```diff
@@ -22,11 +22,12 @@
...
-		"@enact/core": "^4.9.0",
-		"@enact/i18n": "^4.9.0",
+		"@enact/core": "^5.0.0-alpha.1",
+		"@enact/i18n": "^5.0.0-alpha.1",
 		"@enact/sandstone": "../../",
-		"@enact/spotlight": "^4.9.0",
-		"@enact/ui": "^4.9.0",
+		"@enact/spotlight": "^5.0.0-alpha.1",
+		"@enact/ui": "^5.0.0-alpha.1",
+		"@reduxjs/toolkit": "^2.2.2",
```
Comment (MikyungKim) [inline samples/event-logger/package.json:30]: Please remove this.

File: samples/qa-virtualgridlist/package.json [hunk 3]
```diff
@@ -22,11 +22,12 @@
...
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
+    "@enact/i18n": "^5.0.0-alpha.1",
     "@enact/sandstone": "../../",
-    "@enact/spotlight": "^4.9.0",
-    "@enact/ui": "^4.9.0",
+    "@enact/spotlight": "^5.0.0-alpha.1",
+    "@enact/ui": "^5.0.0-alpha.1",
+    "@reduxjs/toolkit": "^2.2.2",
```
Comment (MikyungKim) [inline samples/qa-virtualgridlist/package.json:30]: Please remove this. We've removed this from https://github.com/enactjs/sandstone/pull/1572.

File: samples/qa-virtualgridlistnative-preserve-focus/package.json [hunk 4]
```diff
@@ -22,11 +22,12 @@
...
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
+    "@enact/i18n": "^5.0.0-alpha.1",
     "@enact/sandstone": "../../",
-    "@enact/spotlight": "^4.9.0",
-    "@enact/ui": "^4.9.0",
+    "@enact/spotlight": "^5.0.0-alpha.1",
+    "@enact/ui": "^5.0.0-alpha.1",
+    "@reduxjs/toolkit": "^2.2.2",
```
Comment (MikyungKim) [inline samples/qa-virtualgridlistnative-preserve-focus/package.json:30]: Please remove this. We've removed this from https://github.com/enactjs/sandstone/pull/1572.

File: samples/qa-virtuallist/package.json [hunk 5]
```diff
@@ -22,11 +22,12 @@
...
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
+    "@enact/i18n": "^5.0.0-alpha.1",
     "@enact/sandstone": "../../",
-    "@enact/spotlight": "^4.9.0",
-    "@enact/ui": "^4.9.0",
+    "@enact/spotlight": "^5.0.0-alpha.1",
+    "@enact/ui": "^5.0.0-alpha.1",
+    "@reduxjs/toolkit": "^2.2.2",
```
Comment (MikyungKim) [inline samples/qa-virtuallist/package.json:30]: Please remove this. We've removed this from https://github.com/enactjs/sandstone/pull/1572.

File: .travis.yml (hunk 1 above)
Comment (vJIYEv) [reply .travis.yml]: ok

File: samples/event-logger/package.json (hunk 2 above)
Comment (vJIYEv) [reply samples/event-logger/package.json]: ok

File: samples/qa-virtualgridlist/package.json (hunk 3 above)
Comment (vJIYEv) [reply samples/qa-virtualgridlist/package.json]: ok

File: samples/qa-virtualgridlistnative-preserve-focus/package.json (hunk 4 above)
Comment (vJIYEv) [reply samples/qa-virtualgridlistnative-preserve-focus/package.json]: ok

File: samples/qa-virtuallist/package.json (hunk 5 above)
Comment (vJIYEv) [reply samples/qa-virtuallist/package.json]: ok

**Comment by vJIYEv** [comment]
unit test seems to be failed because of enact version.
It will be ok after merge enact branch `feature/enact_v5` to `develop` in https://github.com/enactjs/enact/pull/3261

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Review by 0x64 — APPROVED** [review]

**Pull request merged**